### DISCUSSION TITLE
Datatable Column Interface Clarification

### DISCUSSION THREAD

**Discussion by JackHimes** [primefaces/community#172] [PrimeVue]
Hello,

I'm currently working on a Vue3 project that is using the Datatable component from PrimeVue. I was hoping to get some clarification on how to use the "ColumnContext" interface, more specifically the [disabled](https://primevue.org/datatable/#api.column.interfaces.ColumnContext.disabled) option. I've look rigorously through the example code as well as online with no luck. Thank you!

Update: I have since found [this](https://github.com/primefaces/primevue/issues/324) GitHub discussion and the consensus appears to be that it does _not_ work, is this correct?

Update 2: The link from the previous update has since been updated, which includes a solution/example of how to workaround this. I hope this helps someone in the future!

**Comment by tugcekucukoglu** [comment]
Hello,

Yes, #324 is still an open enhancement issue. On the other hand, ColumnContext is an object of the options that are available for the Column component of the pt property, which is a completely different concept. `disabled` option of the ColumnContext indicates whether the row to which the column belongs is disabled or not. Its usage is generally as follows:

```
<Column :pt="{
   root: ({ context }) => ({ class: context.disabled ? '...' : '...' })
}">
...
</Column>
```
